### ISSUE TITLE
Overall "GetCapabilities"-extract

### ISSUE THREAD

**Issue by michmuel** [camptocamp/GeoMapFish#87]
### Who requested this new feature?
Canton Basel-Landschaft

### Is your feature request related to a problem?
Context of the feature request: When geodata is available in the form of several qgis projects, and an ogc_server is defined for each qgis project in the geoportal, it is not possible to obtain an overall "GetCapabilities"-extract by a single request.

### Describe the solution you'd like
The "GetCapabilities"-request in Geomapfish 2.8 relates always to one particular ogc_server. The name of the ogc_server is specified by a specific query parameter in the "GetCapabilities"-request:
https://{geoportal}/mapserv_proxy?ogcserver=ogcserver_1&SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.3

We would like to have added the possibility to request an aggregated "GetCapabilities" extract for several of the ogc_servers listed in the geoportal. The advantage of this new feature is that only one url for the WMS and WFS, respectively, must be communicated to the user commnunity. This facilitates using the WMS/WFS provided by the geoportal in desktop GIS.

We propose to create the response to this request from the geoportal database only. The theme and layer structure defined in the database could be taken over and be submitted to a user by using this approach. The layer structure would be identical in WMS/WFS and web client. Map-layers of ogc_servers not described in the database would not be included in the aggregated "GetCapabilities"-request.

### Additional information or points to watch out for
None

**Comment by jwkaltz** [comment]
Idea for implementation with the current Geomapfish framework: the mapproxy software might be used for this purpose.

**Comment by remyguillaume** [comment]
Hi Wolfgang,

I think the problem here is that MapProxy does not support WFS. Do you know how we could configure GMF to use MapProxy but still be able to use WFS?

**Comment by maltaesousa** [comment]
Yes but WFS can be provided by another aggregator. Maybe use mapproxy just for WMS/WMTS aggregation and let's say pygeoapi as an aggregator of OGC API capabilities.  And if we really want old style WFS, it can be aggregated in GMF.

**Comment by michmuel** [comment]
I would appreciate if configuration could be handled in one place, optimally in the c2c geoportal database. Would it not be possible to generate the WMS/WFS get-capabilities-extract from information from the db?
